Items Editing/simon_Kontakt Sample Namer.lua

```diff
@@ -0,0 +1,209 @@
+
+			for l = 1, dyn_amount do -- l corresponds to current dynamic level
+
+				if l == 1 then -- initialize dynamic numbers for each new note group
+					dyn_min = 1
+					dyn_max = dyn_size
+				end
+
+				if l == dyn_amount then
+					dyn_max = 127
+				end
+
+				for k = 0, rr_amount - 1 do -- k loops through sets of rrs and applies dynamic level text to items
+					item = reaper.GetSelectedMediaItem(0, i + k)
+					item_take = reaper.GetActiveTake(item)
+
+					if set_color == true then
+						reaper.SetMediaItemInfo_Value(item,"I_CUSTOMCOLOR", colors[l]) -- Set color
+					end
+
+					local _, name = reaper.GetSetMediaItemTakeInfo_String(item_take, "P_NAME", "RR" .. k, false) -- get previous take name
+					local _, name = reaper.GetSetMediaItemTakeInfo_String(item_take, "P_NAME", name .. "_" .. dyn_min .. "_" .. dyn_max, true) -- set new take name
+					
+
+				end
+
+				dyn_min = dyn_min + dyn_size
+				dyn_max = dyn_max + dyn_size
+				i = i + rr_amount
+
+			end
+		end
+	end
+end
+
+function AppendRegion()
+	count_sel_item = reaper.CountSelectedMediaItems(0)
+	  
+	i = 0
+	  
+	while i < count_sel_item do
+	  
+		item = reaper.GetSelectedMediaItem(0, i)
+		        
+		item_pos = reaper.GetMediaItemInfo_Value(item, "D_POSITION")
+		
+		item_take = reaper.GetActiveTake(item)
+		
+		last_marker, last_region = reaper.GetLastMarkerAndCurRegion(0, item_pos)
+		
+		retval, region_exists, region_pos, region_end, region_name, marker_index = reaper.EnumProjectMarkers(last_region)
+		
+		local _, name = reaper.GetSetMediaItemTakeInfo_String(item_take, "P_NAME",region_name, false) -- get name
+		local _, name = reaper.GetSetMediaItemTakeInfo_String(item_take, "P_NAME", name .. region_name, true) -- store new name
+		
+		i = i + 1
+	end
+end
+
+function ImportTrack()
+	count_sel_item = reaper.CountSelectedMediaItems(0)
+	  
+	i = 0
+	  
+	for i = 0, count_sel_item -1 do
+	  
+		item = reaper.GetSelectedMediaItem(0, i)
+		item_track = reaper.GetMediaItem_Track(item)
+
+		retval, item_track_name = reaper.GetSetMediaTrackInfo_String(item_track, "P_NAME", "", false)
+		
+		item_take = reaper.GetActiveTake(item)
+		
+		local _, name = reaper.GetSetMediaItemTakeInfo_String(item_take, "P_NAME","", false) -- get name
+		local _, name = reaper.GetSetMediaItemTakeInfo_String(item_take, "P_NAME", item_track_name, true) -- store new name
+	end
+end
+
+function Main()
+	colors = {}
+    for i = 1, 20 do
+      colors[i] = 0
+    end
+
+    colors[1] = 25296750
+    colors[2] = 33489024
+    colors[3] = 23167906
+    colors[4] = 28149483
+    colors[5] = 31505564
+    colors[6] = 15853876
+    colors[7] = 31300864
+    colors[8] = 16844908
+    colors[9] = 12952244
+    colors[10] = 1902341
+    colors[11] = 36810826
+    colors[12] = 26194408
+    colors[13] = 2309881
+    colors[14] = 83212661
+    colors[15] = 47235791
+    colors[16] = 4145279
+    colors[17] = 22940017
+    colors[18] = 5309774
+    colors[19] = 42831570
+    colors[20] = 27106875
```

Comment:
Colors are platform-specific. Red and blue are swapped on Windows/Linux vs macOS.

Some of these colors have an invalid enable flag such as 42831570 (0x28d8ed2 instead of 0x18d8ed2), or 1902341 which doesn't have it.

Reply:
> Colors are platform-specific. Red and blue are swapped on Windows/Linux vs macOS.

Could you clarify if this applies to all MacOS versions and if it's Windows/Linux vs MacOS or Windows vs Linux/MacOS as  was stated here https://forums.cockos.com/showthread.php?t=253981#5 ?